nixos/modules/programs/_1password-gui.nix

```diff
@@ -0,0 +1,55 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.programs._1password-gui;
+
+in {
+  options = {
+    programs._1password-gui = {
+      enable = mkEnableOption "The 1Password Desktop application with browser integration";
+      groupId = mkOption {
+        default = 44399;
+        defaultText = "44399";
+        example = "44399";
+        description = "This provides an ID for the onepassword group, which will own the setgid helper applications. This value can be anything, so long as it is unique on the system. 44399 was merely chosen randomly during development.";
+        type = types.ints.positive;
+      };
+    };
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs._1password-gui;
+      defaultText = literalExpression "pkgs._1password-gui";
+      example = literalExpression "pkgs._1password-gui";
+      description = ''
+        The 1Password derivation to use. This can be used to upgrade from the stable release that we keep in nixpkgs to the betas.
+        '';
+    };
+  };
+
+  config = mkIf cfg.enable {
+    environment.systemPackages = [ package ];
```

Comment:
pretty sure this (and the following instances of plain `package`) will not evaluate cleanly. did you mean `cfg.package`?

Reply:
And now that is fixed.